wordpress-mobile/WordPress-FluxC-Android
# [Woo] Add Enable Coupons Function

### Description

This PR allows for enabling Coupons directly via API. This uses the WooCommerce Site Settings Option in https://woocommerce.github.io/woocommerce-rest-api-docs/?shell#update-a-setting-option

### To Test
1. In wp-admin, disable Coupons first in WooCommerce > Settings > General tab > "Enable coupons"
3. Run Example app,
4. Go to Woo > Coupons 
5. Click "Select Site" and pick the site from step 1,
6. Back to Example app, click "Enable Coupons Feature" and check the log for success message,
7. Check back in wp-admin and make sure Coupons is now enabled by going to Marketing > Coupons
